src/components/settings/settings/EditSettingsForm.tsx

```diff
@@ -1085,34 +1085,34 @@ class EditSettingsForm extends Component<IProps, IState> {
                   <H3>
                     {intl.formatMessage(messages.subheadlineFerdiumProfile)}
                   </H3>
-                  <p>
```

Comment:
This was causing a React error in dev tools because its invalid HTML to have a `div` within a `p`